README.md

```diff
@@ -58,11 +58,8 @@ form = HTTP::FormData.create({
 This library aims to support and is [tested against][ci] the following Ruby
 versions:
 
-* Ruby 2.1.x
-* Ruby 2.2.x
-* Ruby 2.3.x
-* Ruby 2.4.x
-* JRuby 9.1.x.x
+* Ruby 2.4+
```

Comment:
After recent incident in http gem I would like to keep this list  explicit

Reply:
Updated 👍